ui/lib/apps/Statement/pages/List/index.tsx

```diff
@@ -105,9 +101,9 @@ export default function StatementsOverview() {
   )
 
   return (
-    <div style={{ height: '100vh', display: 'flex', flexDirection: 'column' }}>
+    <div className={styles.list_container}>
```

Comment:
BTW we prefer using `styles.listContainer`. We have two conventions in the code and we can change it. This is a previously given up task in https://github.com/pingcap/tidb-dashboard/pull/768

Reply:
Oh, wow. I just follow the "code". I also prefer camelCase.